src/utils/device.js

```diff
@@ -123,10 +123,29 @@ module.exports.isMobile = isMobile;
  */
 function isTablet (mockUserAgent) {
   var userAgent = mockUserAgent || window.navigator.userAgent;
-  return /ipad|Nexus (7|9)|xoom|sch-i800|playbook|tablet|kindle/i.test(userAgent);
+
+  var _isTablet = /ipad|Nexus (7|9)|xoom|sch-i800|playbook|tablet|kindle/i.test(userAgent);
```

Comment:
we don't use variable names with underscores. I also see the regular expression is checking for ipad. needed?

Reply:
thanks

Reply:
also wonder if this regex for tablet still provide good coverage. I see device versions hardcoded (e.g `Nexus 7|9`).